docs/technical_reference/unit_models/electroNP_ZO.rst

```diff
@@ -0,0 +1,83 @@
+ElectroNP (ZO)
+====================
+This electrochemical nutrient removal (ElectroNP) unit model
+   * is a zero-order model that enables the user to specify performance in terms of solute rejection
+   * supports a single liquid phase only
+   * supports steady-state only
+   * assumes isothermal conditions
+
+.. index::
+   pair: watertap.unit_models.electroNP_ZO;electroNP_ZO
+
+.. currentmodule:: watertap.unit_models.electroNP_ZO
+
+Degrees of Freedom
+------------------
+The zero-order electroNP model has at least 6 degrees of freedom that should be fixed for the unit to be fully specified.
+Typically, the following variables are fixed:
+
+   * inlet volumetric flow rate
+   * inlet temperature
+   * inlet pressure
+   * electricity intensity
+   * dosage of magnesium chloride per phosphorus removal
+
+There are 1 degree of freedom for each solute in a given property model:
+
+   * inlet solute concentration
+
+Model Structure
+------------------
+This electroNP model is built based on IDAES Separator module. It consists of one inlet port and two outlet ports as
```

Comment:
```suggestion
This electroNP model is built based on the IDAES Separator module. It consists of one inlet port and two outlet ports as
```
And consider making "IDAES Separator" a link to the associated IDAES readthedocs.

Reply:
Good idea, I added the link